metering/reports/metering-storage-locations.adoc

```diff
@@ -51,15 +51,16 @@ spec:
 ----
 <1> (optional) The filesystem URL for Presto and Hive to use for the database. This can be an `hdfs://` or `s3a://` filesystem URL.
 
-There are some additonal optional fields that can be specified in the `hive` section:
+There are some additional optional fields that can be specified in the `hive` section:
 
 * (optional) defaultTableProperties:  Contains configuration options for creating tables using Hive.
 * (optional) fileFormat: The file format used for storing files in the filesystem. See the link:https://cwiki.apache.org/confluence/display/Hive/LanguageManual+DDL#LanguageManualDDL-StorageFormatsStorageFormatsRowFormat,StorageFormat,andSerDe[Hive Documentation on File Storage Format] for a list of options and more details.
 * (optional) rowFormat: Controls the link:https://cwiki.apache.org/confluence/display/Hive/LanguageManual+DDL#LanguageManualDDL-RowFormats&SerDe[ Hive row format]. This controls how Hive serializes and deserializes rows. See the link:https://cwiki.apache.org/confluence/display/Hive/LanguageManual+DDL#LanguageManualDDL-RowFormats&SerDe[Hive Documentation on Row Formats and SerDe] for more details.
 
 == Default StorageLocation
-If an annotation `storagelocation.metering.openshift.io/is-default` exists and is set to `true` on a StorageLocation resource, then that resource becomes the default storage resource.  Any components with a storage configuration option where StorageLocation is not specified will use the default storage resource. There can only be on default storage resource. If more than one resource with the annotation exists, an error will be logged and the operator will consider there to be no default.
+If an annotation `storagelocation.metering.openshift.io/is-default` exists and is set to `true` on a StorageLocation resource, then that resource becomes the default storage resource.  Any components with a storage configuration option where StorageLocation is not specified will use the default storage resource. There can be only one default storage resource. If more than one resource with the annotation exists, an error is logged and the operator considers there to be no default.
```

Comment:
s/operator/Operator/
Minor nit, is there another way to say that "considers there to be"? It's throwing me off. I don't know if this is correct to say but maybe something like:

> If more than one resource with the annotation exists, an error is logged because the Operator cannot determine the default.

Reply:
> Minor nit, is there another way to say that "considers there to be"? It's throwing me off. I don't know if this is correct to say but maybe something like:
> 
> "If more than one resource with the annotation exists, an error is logged because the Operator cannot determine the default."

@timflannagan1 Can you PTAL at @bobfuru's wording suggestion. Thank you!

Reply:
@lbarbeevargas that suggestion works for me :+1: